docs/diagrams/CommandsDiagram.puml

```diff
@@ -0,0 +1,59 @@
+@startuml
+!include Styles.puml
+
+Package "Commands                    "  <<Rectangle>>{
+     Class "{abstract}\nCommand" as Command COLOR_COMMANDS {
+     {method} + setData
+     {method} + setParams
+     {method} + execute: String
+     }
+     Class ViewCommand COLOR_COMMANDS
+     Class EditInstrumentCommand COLOR_COMMANDS
+     Class DoneCommand COLOR_COMMANDS
+     Class DeleteCommand COLOR_COMMANDS
+
+     Class InvalidCommand COLOR_COMMANDS
+     Class ABCCommand COLOR_COMMANDS
```

Comment:
Can add a note on what ABC is a placeholder for

Reply:
ok